### DISCUSSION TITLE
Dynamic/Repeater field issue

### DISCUSSION THREAD

**Discussion by golatam** [orchidsoftware/platform#1908] [Q&A]
Is it possible to create a dynamic field in Orchid?

For example, to have a field of the text type and be able to make any number of duplicates of this field?

I found the library https://github.com/Nks/orchid-repeater-field/
But it doesn't work correctly, all my attempts to investigate it gave no result

Thank you!

**Comment by modmatrix** [comment]
+1. I would love to adopt Orchid, but the lack of a reliable repeater field is a deal-breaker.

This should absolutely be standard, out-of-the-box functionality.

**Comment by modmatrix** [comment]
I made the same request a few months ago:

https://github.com/orchidsoftware/platform/discussions/1964
